api_app/models/domain/events.py

```diff
@@ -1,13 +1,12 @@
-from typing import List
+from typing import Dict
 from models.domain.azuretremodel import AzureTREModel
 
 
 class AirlockNotificationData(AzureTREModel):
     request_id: str
     event_type: str
     event_value: str
-    researchers_emails: List[str]
-    owners_emails: List[str]
+    notification_emails: Dict
```

Comment:
nit `notification` is implicit from the class name, plain emails?

Reply:
Fixed